packages/core/admin/admin/src/reducer.ts

```diff
@@ -29,7 +29,8 @@ const THEME_LOCAL_STORAGE_KEY = 'STRAPI_THEME';
 const LANGUAGE_LOCAL_STORAGE_KEY = 'strapi-admin-language';
 
 export const getStoredToken = (): string | null => {
-  const token = localStorage.getItem(STORAGE_KEYS.TOKEN) ?? getCookieValue(STORAGE_KEYS.TOKEN);
+  const token =
+    localStorage.getItem(STORAGE_KEYS.TOKEN) ?? JSON.stringify(getCookieValue(STORAGE_KEYS.TOKEN));
```

Comment:
I think this could result in the token being the string "null" if getCookieValue return null. Is that intended / could cause issues?

Reply:
it shouldn't be null, and if it is then the users operation wouldn't work and they would be logged out which removes the cookie